src/Helpers/helperUploadPath.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if (!function_exists('getUploadPath')) {
+
+    /**
+     * Return upload path.
+     *
+     * @param string $slug
+     * @param string $path
+     *
+     * @return string
+     */
+    function getUploadPath($slug, $path)
+    {
+        $_dirs = explode('/', $path);
```

Comment:
Variable naming is too generic.  Also, $_ is generally reserved for PHP superglobals.